### ISSUE TITLE
Cannot run OBS

### ISSUE THREAD

**Issue by slashtechno** [chromebrew/chromebrew#6272]
Provide a short description of your problem here.
I tried to install OBS on my Chromebook and could not run it.
If you executed a specific command:
```shell
crew install obs
obs
```

Output:
```
obs: Free and open source software for video recording and live streaming.
https://obsproject.com/
Version: 24.0.5
License: GPL-2
Performing pre-flight checks...
Precompiled binary available, downloading...
################################################################################################################ 100.0%
Obs archive downloaded.
Unpacking archive using 'tar', this may take a while...
Performing pre-install...
Performing install...
/usr/local/sbin/ldconfig: /usr/local/lib64/libQt5Svg.so.5 is not a symbolic link

Performing post-install for obs...

To get started, type 'obs'.

To completely remove, execute the following:
crew remove obs
rm -rf ~/.config/obs-studio/

Obs installed!
```
Output of `obs`:
```bash
obs: error while loading shared libraries: libQt5X11Extras.so.5: cannot open shared object file: No such file or direct
```

## Versions and system information
Chromebrew: `1.16.9`

ChromeOS: `CHROMEOS_RELEASE_BUILD_NUMBER=14092`

Architecture: `x86_64`

## Additional information
The first time, I used `crew reinstall obs` since for previous program installs, `crew install` did not work. 
What I think needs to be done:
I think during the install, a symbolic link needs to be made or the library installed.

**Comment by slashtechno** [comment]
> The following command should fix it:
> ```shell
> crew install qtx11extras
> ```

Thank you.

**Comment by necrolatry** [comment]
the lib64 folder is empty in the package from gitlab, just had a look in my lib64 folder and libQt5X11Extras.so.5 is missing, tried crew install -s qtx11extras and it wouldn't build say the package is broken

**Comment by necrolatry** [comment]
Yeh that's because there are no librarys in the libQt5X11Extras tar.xz I'm having the same issue,

Try to build the source crew install -s qtx11extras
you might have better luck than me because your on ChromeOS not Cloudready

In cloudready it throws Error: There is a conflict with the same file in another package.

qtsvg./usr/local/share/Qt-5/src/src.pro package conflicts below:
qtsvg./usr/local/share/Qt-5/src/Makefile
Exiting due to above errors.

Basically I need to find the conflicting package to get this to build which should be fun, maybe another night cheers.

**Comment by necrolatry** [comment]
Had a bit of a play around with this and may have found a solution to get it loading (Cloud Ready again)

crew reinstall qtx11extras
download this, extract the missing libs into lib64, https://drive.google.com/file/d/1AoBhWAtLJ1rdaUL7_zDNLnROBqkS885h/view?usp=sharing
crew reinstall qtbase libxcb libxinerama xcb_util qtwayland qtsvg
libtool --finish /usr/local/lib64
(fixes the problem here too #5350  )

obs runs but throws an error loading obs_x264
attempted build from source - no good
attempted portable build from github - success needed deps below
crew install pipewire pciutils

read on a few forums that if the plugin path gets changed from the default /usr/lib to /usr/local/lib64 it can't find the plugin folder so throws the plugin error and crashes, have no idea how to change that to fix the package, maybe somebody here knows?

**Comment by yeesterPlus** [comment]
cannot install!
